docs/release/3.3/release-3-3-33-2.md

```diff
@@ -0,0 +1,95 @@
+OSG Software Stack -- Data Release -- 3.3.33-2
+==============================================
+
+**Release Date**: 2018-04-05
+
+Summary of changes
+------------------
+
+This release contains:
+
+-   CA Certificates based on [IGTF 1.90](http://dist.eugridpma.info/distribution/igtf/current/CHANGES)
+    -   Added new Grid-FR hierarchy for Renater (AC-GRID-FR series) (FR)
+    -   Added new GARUDAINDIA2 root for key roll-over IGCA (IN)
+    -   Updated contact metadata for UNAM trust anchors (MX)
+-   [VO Package v78](https://github.com/opensciencegrid/osg-vo-config/releases/tag/release-78)
+    -   Update ATLAS default mappings
+
+These [JIRA tickets](https://jira.opensciencegrid.org/issues/?jql=project%20%3D%20SOFTWARE%20AND%20fixVersion%20%3D%203.3.33-2%20ORDER%20BY%20priority%20DESC%2C%20key%20DESC) were addressed in this release.
+
+Detailed changes are below. All of the documentation can be found [here](../../).
+
+Updating to the new release
+---------------------------
+
+### Update Repositories
+
+To update to this series, you need [install the current OSG repositories](../../common/yum#install-osg-repositories).
```

Comment:
"you need to install" maybe?

And also, could you make this a site relative link? e.g. `/common/yum#install-osg-repositories`

Reply:
OK, Let's handle that with a bulk change later.